core-activities.md

```diff
@@ -1,16 +1,16 @@
 # Core Activities
 
-## Implementation and Decommissioning
-We lead strategy for implementation and decommissioning of systems related to archival functions. We evaluate workflows, systems, and processes holistically, with an eye towards removing redundancy and eliminating unnecessary manual intervention so that our colleagues are better able to use their unique skills in acquiring, preserving, and providing access to archival collections.
+## Implement and Develop Systems
+We lead strategy for the selection, implementations, and decommissioning of systems key to archival functions. We evaluate workflows, systems, and processes holistically, with an eye towards removing redundancy and eliminating unnecessary manual intervention so that our colleagues are better able to use their unique skills in acquiring, preserving, and providing access to archival collections. We develop systems that fit the RAC's needs and use cases, as well as work with open source applications and their communities.
 
-## User Experience Design
-We support and advocate for the usability of archival systems and processes at the RAC by taking a user-centered approach to design and implementation. We strive to improve user experience by engaging users throughout all stages of the implementation and systems lifecycle. We support this work through user research, performing user experience testing, and involving stakeholders throughout the process. We recognize that accessibility is integral to usability, and are committed to ensuring that people with disabilities have access to our systems and content. We look to the W3C Web Content Accessibility Guidelines 2.0 (Level AA) as the target standard for meeting this commitment.
+## Design for User Experience
+We advocate for and improve the usability of archival systems and processes at the RAC by taking a user-centered approach to design and implementation. We strive to improve user experience by engaging users throughout all stages of the implementation and systems lifecycle. We perform user research, create and lead user experience tests, and involve stakeholders throughout the design process. We recognize that accessibility is integral to usability, and are committed to ensuring that people with disabilities have access to our systems and content.
```

Comment:
```suggestion
We advocate for and improve the usability of archival systems and processes at the RAC by taking a user-centered approach to design and implementation, striving to improve user experience by engaging users throughout all stages of the systems lifecycle. We perform user research, create and lead user experience tests, and involve stakeholders throughout the design process. Because we recognize that accessibility is integral to usability, we are committed to ensuring that people with disabilities have access to our systems and content.
```